SPECS/xxhash/xxhash.spec

```
@@ -0,0 +1,186 @@
+Name:     xxhash
+Version:  0.8.1
+Release:  3%{?dist}
+Summary:  Extremely fast hash algorithm
+#    The source for the library (xxhash.c and xxhash.h) is BSD
+#    The source for the command line tool (xxhsum.c) is GPLv2+
+License:  BSD and GPLv2+
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:      http://www.xxhash.com/
+Source0:  https://github.com/Cyan4973/xxHash/archive/refs/tags/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
+#    Fix compilation on RHEL 7 ppc64le (gcc 4.8)
+#    https://github.com/Cyan4973/xxHash/issues/622
+#    https://github.com/Cyan4973/xxHash/pull/631
+Patch0:   %{name}-epel7-ppc64le.patch
+
+BuildRequires:	make
+BuildRequires:	gcc
+BuildRequires:	doxygen
+
+%description
+xxHash is an Extremely fast Hash algorithm, running at RAM speed
+limits. It successfully completes the SMHasher test suite which
+evaluates collision, dispersion and randomness qualities of hash
+functions. Code is highly portable, and hashes are identical on all
+platforms (little / big endian).
+
+%package libs
+Summary:	Extremely fast hash algorithm - library
+License:	BSD
+
+%description libs
+xxHash is an Extremely fast Hash algorithm, running at RAM speed
+limits. It successfully completes the SMHasher test suite which
+evaluates collision, dispersion and randomness qualities of hash
+functions. Code is highly portable, and hashes are identical on all
+platforms (little / big endian).
+
+%package devel
+Summary:  Extremely fast hash algorithm - development files
+License:  BSD
+Requires:  %{name}-libs%{?_isa} = %{version}-%{release}
+# By setting XXH_INLINE_ALL, xxhash may be used as a header-only library.
+# Dependent packages that use xxhash this way must BR this virtual Provide:
+Provides:  %{name}-static = %{version}-%{release}
+
+%description devel
+Development files for the xxhash library
+
+%package doc
+Summary:  Extremely fast hash algorithm - documentation files
+License:  BSD
+BuildArch:  noarch
+
+%description doc
+Documentation files for the xxhash library
+
+%prep
+%autosetup -p1 -n xxHash-%{version}
+
+%build
+# Enable runtime detection of sse2/avx2/avx512 on intel architectures
+%ifarch %{ix86} x86_64
```

Nit: We can simplify this, since we don't support %{ix86} arches

```suggestion
%ifarch x86_64
```

---

Removed.
Verified the rpm build with changes locally.